### DISCUSSION TITLE
How to allow submit even if there are errors ?

### DISCUSSION THREAD

**Discussion by erkanisuf** [react-hook-form/react-hook-form#8244] [Q&A]
Hello. I want to create form that will allow me to make submit even there are errors in the inputs. I am trying to achieve that when user press Submit button , to get confirmation window that says 

> "Fields: Input1,Input2 (basically list  that says what is empty) .... ,  are empty , would you like to continue?" . 

 Was thinking to use for each input {required...} , and on submit to use the error tracking as just "checker" what is empty.

**Comment by bluebill1049** [comment]
Just use get values API
